web/src/lightbox.js

```diff
@@ -479,7 +483,7 @@ export function parse_media_data(media) {
     if (is_compose_preview_media) {
         sender_full_name = people.my_full_name();
     } else {
-        const $message = $parent.closest("[zid]");
+        const $message = $parent.closest("[id^='zfilt']");
```

Comment:
Can you add a comment explaining why you chose that selector? Might be better to make this change in its own commit.